Before: b707047b487c68f2fe8c2d1ac6c4636fcf2650562127f283339cc99762e9cec6
After: ecb07501b873ab031a49801b6b9f9237a53a2b3d9a70dec2d1bb3521975247ea
Patch: 9c92bbc595b40f7e0f0e21c2d49b68f176caa4bed133cd9f07dce8f46f646d2d
--- corvid-features/corvid.rb	2012-07-31 15:43:45.842682625 +1000
+++ corvid-features/corvid.rb	2012-07-31 15:43:51.289376468 +1000
@@ -14,11 +14,4 @@
   copy_executable 'bin/yardoc'
   copy_executable 'bin/yri'
   empty_directory 'lib'
-#  empty_directory 'wtffff'
-end
-
-def update(ver)
-  if ver == 2
-    empty_directory 'wtffff'
-  end
 end
